LS-457-starlight-doc (#35)
datastax/pulsar-docs: modules/ROOT/pages/starlight.adoc

= Starlight for Luna Streaming

Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.

== Starlight for RabbitMQ

> Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.

Reviewer: ```suggestion
Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Pulsar streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
```

> Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 

Reviewer: Does this apply to Starlight for JMS? What about when Starlight for Kafka is added? If all of the Starlight connectors are included in the support plan, let's move this to the first section and make it more generic. Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.

Reviewer: @mendonk Let's also link to the support policy or another Luna Streaming reference similar. Probably https://www.datastax.com/legal/luna-streaming-supplemental-support-policy, but confirm with @chrislatimer.

> For more information on Starlight for Rabbit MQ, check out 
> https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].

Reviewer: ```suggestion
For more information on Starlight for RabbitMQ, check out 
https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[getting started with Starlight for RabbitMQ].
```

== Starlight for JMS

> Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Apache Pulsar™ is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.

Reviewer: ```suggestion
Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Pulsar is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.
```

> To get started with Starlight for JMS in Luna Streaming, check out 
> https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS Quickstart].

Reviewer: ```suggestion
To get started with Starlight for JMS in Luna Streaming, check out 
https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS installation].
```

> = Starlight for Luna Streaming
> 
> Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.
> 
> Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 
> 
> For implementation with JMS or Kafka on a Pulsar cluster, we also offer Starlight for JMS and Starlight for Kafka 
> 
> == Starlight for RabbitMQ
> 
> Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar.
> 
> For more information on Starlight for Rabbit MQ, check out 
> https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].

Reviewer: ```suggestion
= Starlight for Luna Streaming

Starlight for Luna Streaming allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform with various connectors for Apache Pulsar™.

== Starlight for RabbitMQ

Starlight for RabbitMQ™ combines the industry-standard AMQP 0.9.1 (RabbitMQ) API with the cloud-native and horizontally scalable Apache Pulsar™ streaming platform, providing a powerful way to modernize your RabbitMQ infrastructure, improve performance, and reduce costs.

Starlight for RabbitMQ™ is open source and is included in DataStax’s Luna Streaming Enterprise support for Pulsar. 

For more information on Starlight for Rabbit MQ, check out 
https://docs.datastax.com/en/starlight-rabbitmq/docs/1.0/getting-started.html[Starlight for Rabbit MQ Quickstart].
```

== Starlight for JMS

Starlight for JMS allows enterprises to take advantage of the scalability and resiliency of a modern streaming platform to run their existing JMS applications. Because Apache Pulsar™ is open-source and cloud-native, Starlight for JMS enables enterprises to move their JMS applications to run on-premises and in any cloud environment.

To get started with Starlight for JMS in Luna Streaming, check out 
https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-install.html[Starlight for JMS Quickstart].

> == Starlight for Kafka
> 
> Starlight for Kafka allows your Apache Kafka® clients to use an Apache Pulsar® cluster.
> 
> To get started with Starlight for Kafka in Luna Streaming, check out the https://github.com/datastax/starlight-kafka-docs[Starlight for Kafka repo]. 

Reviewer: ```suggestion
// == Starlight for Kafka
// 
// Starlight for Kafka allows your Apache Kafka® clients to use an Apache Pulsar® cluster.
// 
// To get started with Starlight for Kafka in Luna Streaming, check out the https://github.com/datastax/starlight-kafka-docs[Starlight for Kafka repo]. 
```
